### PR TITLE
9 UI

### PR DESCRIPTION
**Pull request by missionStrawberry**
## Feature: GUI implementation, and initializing and calling all use cases.
Description of feature:  
- Make instances of all necessary classes for each use case in Main.
- All GUI screens are created for each use case and the shared main week view. 
- All use cases can be called from WeekViewScreen and are given all necessary information.
### Important Notes:
- Currently WeekViewScreen does not reflect the current state of the program, rather it shows what it would look like if events and tasks were populated without actually making tasks and events for each. 
- Please review the files specific to your use case. 
- The old UI PR had to be closed because its branch got ruined by accident and a new branch (thus this PR) needed to be created. Refer to the [old PR](https://github.com/CSC207-2022F-UofT/course-project-blue-guppies/pull/40) for further information.

### CODE REVIEW

**Review by Fard-Faru — APPROVED** [review]
Looks Good in terms of Complete and Create Task Use Case. Good work Anna!

File: src/main/java/Main.java [hunk 1]
```diff
@@ -1,7 +1,116 @@
...
+        CreateTaskOutputBoundary createTaskOutputBoundary = new CreateTaskPresenter();
+        CreateTaskDsGateway createTaskDsGateway = new CreateTaskDataAccess();
+        TaskFactory taskFactory = new TaskFactory();
+        CreateTaskInputBoundary createTaskInputBoundary = new CreateTaskInteractor(taskFactory,
+                createTaskOutputBoundary, createTaskDsGateway);
+        CreateTaskController createTaskController = new CreateTaskController(createTaskInputBoundary);
+
+        // Initialize modify task classes
+        ModifyTaskOutputBoundary modifyTaskPresenter = new ModifyTaskPresenter();
+        ModifyTaskDataAccess dsGateway = new ModifyTaskDataAccess();
```
Comment (POPPININ) [inline src/main/java/Main.java:52]: This should be a ModifyTaskDsGateway instance, although it is initialized with an object of type ModifyTaskDataAccess.

**Review by POPPININ — CHANGES_REQUESTED** [review]
One minor fix required Anna. Looks good otherwise!

File: src/main/java/Main.java (hunk 1 above)
Comment (missionStrawberry) [reply src/main/java/Main.java]: Thanks for catching that. Fixed it.

**Review by POPPININ — APPROVED** [review]
Thanks for fixing the issue with the dsGateway for Task Modification being explicitly initialized as a Data Access object.

**Review by DanielLivshits — APPROVED** [review]
Hello Anna, 
Everything seems to be in order for the modify event use case. Thank you setting up the UI.

**Review by scissorkhizer — APPROVED** [review]
Everything looks great regarding the initialization of delete event within main, thanks for handling the implementation of UI Anna!

**Review by eatingcorn — APPROVED** [review]
Clear all and the delete task use cases in main looks good. Thanks for implementing UI!

**Pull request merged**